sql/sp_instr.cc

```diff
@@ -27,8 +27,8 @@
 #include "sql_parse.h"    // check_table_access
 #include "sql_prepare.h"  // reinit_stmt_before_use
 #include "transaction.h"  // trans_commit_stmt
-#include "sql_audit.h"
 #include "debug_sync.h"   // DEBUG_SYNC
+#include "sql_audit.h"
```

Comment:
Any reason why we changed header inclusion order here?

Reply:
`sql_audit.h` was added by Percona and above change keeps code more similar to upstream.